src/components/TodoHeader.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="flex justify-between">
+    <h1 class="text-7xl font-bold">To do List</h1>
+    <PlusCircleIcon @click="$emit('addTodo')" class="h-16 w-16 text-green-400 cursor-pointer" />
```

Comment:
Please use a separate function for emit, for instance:
`@click="addTodo"`
`function addTodo() {
   emit('addTodo')
}`